pytest_tests/testsuites/load/test_load.py

```diff
@@ -22,53 +23,61 @@
     DELETERS,
     LOAD_NODE_SSH_PRIVATE_KEY_PATH,
     LOAD_NODE_SSH_USER,
-    LOAD_NODES,
-    LOAD_NODES_COUNT,
+    LOAD_NODE,
     LOAD_TIME,
     LOAD_TYPE,
     OBJ_COUNT,
     OBJ_SIZE,
     OUT_FILE,
     READERS,
-    STORAGE_NODE_COUNT,
+    STORAGE_NODES_COUNT,
     WRITERS,
+    REGION,
 )
 from neofs_testlib.hosting import Hosting
 
+import logging
+logger = logging.getLogger("NeoLogger")
+
 ENDPOINTS_ATTRIBUTES = {
     "http": {"regex": HTTP_GATE_SERVICE_NAME_REGEX, "endpoint_attribute": "endpoint"},
     "grpc": {"regex": STORAGE_NODE_SERVICE_NAME_REGEX, "endpoint_attribute": "rpc_endpoint"},
-    "s3": {"regex": S3_GATE_SERVICE_NAME_REGEX, "endpoint_attribute": "endpoint"},
+    "s3": {"regex": S3_GATE_SERVICE_NAME_REGEX, "endpoint_attribute": "endpoint_data\d"},
 }
 
 
 @pytest.mark.load
 class TestLoad(ClusterTestBase):
-    @pytest.fixture(autouse=True)
-    def clear_cache_and_data(self, hosting: Hosting):
-        clear_cache_and_data(hosting=hosting)
-        yield
-        start_stopped_nodes()
+    # @pytest.fixture(autouse=True)
```

Comment:
How long load will work until we exaust all resources? 
In general dev environment we don't have much. 
If we don't this code then we must delete it :)